docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,374 @@ Example output:
+        8 : 0x8C126E92B82BCF9769A9EDA7CAE05FCB2A15A0EB
+        9 : 0xD0BCA06196EC06094307F435D8788E9426D37166
+        10: 0x881A056121204AAD298010A64011E97E8F63E31C
+        11: 0x0000000000000000000000000000000000000000
+        12: 0x0000000000000000000000000000000000000000
+        13: 0x0000000000000000000000000000000000000000
+        14: 0x77DB66D60AA0C2CD1CEA6C34FED4F2AA014BF285
+        15: 0x0000000000000000000000000000000000000000
+        16: 0x0000000000000000000000000000000000000000
+        17: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        18: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        19: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        20: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        21: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        22: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        23: 0x0000000000000000000000000000000000000000
+    sha256:
+        0 : 0x398991DE8A69685E63E5BC5846CA402CB2F4AE46E89AB977AE9DE6F25D28320F
+        1 : 0xE7F444390126872E822F9006423EBB22FD4A340A95BBA4E98EB354EEAC828BEF
+        2 : 0x884314F10588F807B343C2F23F292E6F748ED4F555FBB5004DBFBF2387E161AA
+        3 : 0x3D458CFE55CC03EA1F443F1562BEEC8DF51C75E14A9FCF9A7234A13F198E7969
+        4 : 0x368F85B3013041DFE203FAAA364F00B07C5DA7B1E5F1DBF2EFB06FA6B9BD92DE
+        5 : 0xB33E1B199263B8FF8644BC166398CCAD03FB5F03FC15CF7A06C1DD73F7E87239
+        6 : 0x3D458CFE55CC03EA1F443F1562BEEC8DF51C75E14A9FCF9A7234A13F198E7969
+        7 : 0xB926225AC488E9C50EF2FA815AA7104B385A06907093BFB1DC62EEB7ABECDDF1
+        8 : 0x7A9EBF0BE340214249EE6025BE1F6093AEDDCFC80A2DF75023B7758BF2E0EF31
+        9 : 0x696B6D9CFC3CCB1520592E1007C0B655EF3AA6C6151823AC944AB4BAFB91743C
+        10: 0x93CA5EEDAD309A46EE27BF7E11C962403B31CB3985B72EC9EAC6997F419F4692
+        11: 0x0000000000000000000000000000000000000000000000000000000000000000
+        12: 0x0000000000000000000000000000000000000000000000000000000000000000
+        13: 0x0000000000000000000000000000000000000000000000000000000000000000
+        14: 0xE3991B7DDD47BE7E92726A832D6874C5349B52B789FA0DB8B558C69FEA29574E
+        15: 0x0000000000000000000000000000000000000000000000000000000000000000
+        16: 0x0000000000000000000000000000000000000000000000000000000000000000
+        17: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        18: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        19: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        20: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        21: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        22: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+        23: 0x0000000000000000000000000000000000000000000000000000000000000000
+    ```
+
+## TPM004.001 PCRALLOCATE Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that `PCRALLOCATE` function works properly.
```

Comment:
And what is that function responsible for?

Reply:
Updated